src/Microsoft.NET.Test.Sdk.targets

```diff
@@ -23,6 +23,12 @@
     <GeneratedProgramFile Condition="'$(GeneratedProgramFile)' ==''">$(IntermediateOutputPath)$(MSBuildProjectName).Program$(DefaultLanguageSourceExtension)</GeneratedProgramFile>
     <GenerateProgramFile Condition="'$(GenerateProgramFile)' == ''">true</GenerateProgramFile>
   </PropertyGroup>
+
+  <!-- Output type for dotnet core test project should be exe. For full CLR it is Library by default. -->
```

Comment:
This is a workaround for sdk bug. Can you please mention the bug here for tracking?

I guess the only caveat is all comparisons in the test csproj will return $(OutputType) to be Library since that's the default, and the targets are imported at the end.